### PR TITLE
LPS-200095 Improve accessibility of Page Audit labels

### PR DESCRIPTION
**Pull request by claraizquierdo**
Motivation
=======
[Link to story or ticket](https://liferay.atlassian.net/browse/LPS-200095)

### CODE REVIEW

**Comment by liferay-echo** [comment]
Thank you for submitting this PR. 👏

Make sure it follows the [Echo Pull Request Template](https://gist.github.com/liferay-echo/5a4455285ff1800f9da146816d85e67d#file-pull-request-template-md) if you haven't already done so. 

Learn more about it [here](https://liferay.atlassian.net/wiki/spaces/ENGECHO/pages/2027455014/Echo+Pull+Request+Etiquette).

File: modules/apps/layout/layout-reports-web/src/main/resources/META-INF/resources/js/components/render_times/FragmentList.tsx [hunk 1]
```diff
@@ -125,6 +125,23 @@ export default function FragmentList({
 			type: SET_SELECTED_ITEM,
 		});
 
+	const calculateLabelsText = (
+		type: string,
+		fromMaster: boolean,
+		chached: boolean
```
Comment (veroglez) [inline modules/apps/layout/layout-reports-web/src/main/resources/META-INF/resources/js/components/render_times/FragmentList.tsx:131]: I think there is a typo here 😅 (chached -> cached)

File: modules/apps/layout/layout-reports-web/src/main/resources/META-INF/resources/js/components/render_times/FragmentList.tsx [hunk 2]
```diff
@@ -125,6 +125,23 @@ export default function FragmentList({
...
+	) => {
+		const labels = [type];
```
Comment (veroglez) [inline modules/apps/layout/layout-reports-web/src/main/resources/META-INF/resources/js/components/render_times/FragmentList.tsx:133]: For this, we can simplify the code in this way: 
```
return [
   type,
   ...(fromMaster ? [Liferay.Language.get('from-master'] : [],
   ...(cached ? [Liferay.Language.get('cached'] : [],
].join(', ');
```

Feel free to choose the way you want 😄

File: modules/apps/layout/layout-reports-web/src/main/resources/META-INF/resources/js/components/render_times/FragmentList.tsx [hunk 3]
```diff
@@ -125,6 +125,23 @@ export default function FragmentList({
...
+
+		if (fromMaster) {
+			labels.push(Liferay.Language.get('from-master'));
+		}
+		if (chached) {
+			labels.push(Liferay.Language.get('cached'));
+		}
+
+		return labels.join(',');
```
Comment (veroglez) [inline modules/apps/layout/layout-reports-web/src/main/resources/META-INF/resources/js/components/render_times/FragmentList.tsx:142]: Here we have to add a space after the comma for better reading

File: modules/apps/layout/layout-reports-web/src/main/resources/META-INF/resources/js/components/render_times/FragmentList.tsx [hunk 4]
```diff
@@ -270,18 +287,29 @@ export default function FragmentList({
 							</p>
 
 							<p className="mb-0">
-								<ClayLabel displayType="secondary">
+								<span className="sr-only">
+									{calculateLabelsText(
```
Comment (veroglez) [inline modules/apps/layout/layout-reports-web/src/main/resources/META-INF/resources/js/components/render_times/FragmentList.tsx:291]: Maybe we can use a name more standard, something like `getLabelText()`

**Comment by veroglez** [comment]
ci:forward

**Comment by liulukiwi** [comment]
See https://github.com/brianchandotcom/liferay-portal/pull/143173

**Comment by liulukiwi** [comment]
ci:close

**Pull request closed without merging**